lib/saxerator/builder/xml_builder.rb

```diff
@@ -1,5 +1,11 @@
+require 'rexml/document'
+
 module Saxerator
   module Builder
+    class XmlDocument < REXML::Document
```

Comment:
Why is there a need for a `to_xml` alias? Less might be more here - meaning we just return a bare REXML::Document and let end-users transform it however they prefer.